### ISSUE TITLE
World Analyzer (Error)

### ISSUE THREAD

**Issue by Muraysi** [Razzmatazzz/RemnantSaveGuardian#192]
![2023-11-18_11h39_35](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/31324029/fc6fbc11-02d5-4f0d-8bd3-dcf51d0bfa54)
I am currently experiencing issues while trying to use my character information. After importing the information, I keep receiving an error message which causes the application to buffer continuously. As a result, I am unable to add, modify, or remove any data. The application remains stuck in a loading loop and if I attempt to backup, it automatically closes. Whenever I reopen it, the same issue recurs. Therefore, I am reaching out to you for assistance. I hope there is a solution to fix this problem.

**Comment by TheNasbit** [comment]
What platform you're using? Epic or Steam? 

Please check the Settings tab, if the Save Folder is correct, and has actual the save files inside.

**Comment by Muraysi** [comment]
> What platform you're using? Epic or Steam?
> 
> Please check the Settings tab, if the Save Folder is correct, and has actual the save files inside.

Steam

**Comment by TheNasbit** [comment]
> What exactly do I need to do?

Can u zip the .sav files inside of the save-folder and post them here?

**Comment by Muraysi** [comment]
> Can u zip the .sav files inside of the save-folder and post them here?
[RemnantSaveGuardian (Error).zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13403892/RemnantSaveGuardian.Error.zip)

Sure! Did I do it correctly?

**Comment by TheNasbit** [comment]
> [RemnantSaveGuardian (Error).zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13403892/RemnantSaveGuardian.Error.zip)
> Sure! Did I do it correctly?

Nah, you zipped the SaveGuardian. We need the content from "C:\Users\<User>\Saved Games\Remnant 2\Steam\<SteamID>\" - inside there zip all the *.sav files.
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/52046382/68e6e98a-6b4d-4e1a-8125-3527e53f3555)

**Comment by Muraysi** [comment]
> > [RemnantSaveGuardian (Error).zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13403892/RemnantSaveGuardian.Error.zip)
> > Sure! Did I do it correctly?
> 
> Nah, you zipped the SaveGuardian. We need the content from "C://Users/< User >/Saved Games/Remnant 2/Steam/< SteamID >/" - inside there zip all the *.sav files. ![image](https://user-images.githubusercontent.com/52046382/284077104-68e6e98a-6b4d-4e1a-8125-3527e53f3555.png)

Okay so this is zipped from here: C:\Users\alaab\Saved Games\Remnant2\Steam\160505827
[profile.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13403931/profile.zip)

And this is zipped from here: C:\Users\alaab\Saved Games\Remnant2\Steam\1509904649\160505827
[save_1.zip](https://github.com/Razzmatazzz/RemnantSaveGuardian/files/13403933/save_1.zip)

Hopefully this time I got it right for you.

**Comment by TheNasbit** [comment]
The manager is just reading from the first one - as it is in the settings tab configured - I wonder why you have the second path at all - huh.
I will see if I find the problem. Thanks.

**Comment by Muraysi** [comment]
> The manager is just reading from the first one - as it is in the settings tab configured - I wonder why you have the second path at all - huh. I will see if I find the problem. Thanks.

Sounds good. I will keep this page open and wait for your response. I am looking forward to the fix!

**Comment by Muraysi** [comment]
I fixed it. It is working fine now :)
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/31324029/1da1acd9-c578-4dd4-adef-890a84f3c2f9)

**Comment by TheNasbit** [comment]
> I fixed it. It is working fine now :) ![image](https://user-images.githubusercontent.com/31324029/284098981-1da1acd9-c578-4dd4-adef-890a84f3c2f9.png)

May I ask "how" u fixed it now? What was the problem?
DId not had the time to work on it sadly until now

**Comment by Muraysi** [comment]
Here is the difference that was causing me an issue:
![2023-11-20_18h10_31](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/31324029/87b70a39-901a-4e77-8527-52c52f7de2ee)
![2023-11-20_18h06_52](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/31324029/8cb1ac76-f699-47d5-aa2b-54fdf1edd2a9)

Speaking of the first image, when the folder was selected, there was another folder 📁to be opened and that is probably why it wasn't reading it correctly.

**Comment by TheNasbit** [comment]
> Speaking of the first image, when the folder was selected, there was another folder 📁to be opened and that is probably why it wasn't reading it correctly.

So basically the program used the folder for a different steam profile that was used on your PC

**Comment by Muraysi** [comment]
![image](https://github.com/Razzmatazzz/RemnantSaveGuardian/assets/31324029/4ff920fa-69ca-4a4a-8c1d-21c447256b6c)
Correct just like this
